<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_showMat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CalibrateImagePath">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SaveResultPath">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoadImage">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>加载图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveResult">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartCalibrate">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AppraiseCalibrate">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>评价标定</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_CameraInParam">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>内参矩阵</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_DistortionParam">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>畸变系数</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
